project_setup/Skeletonize-existing-repository.md

```diff
@@ -0,0 +1,84 @@
+# Skeletonize an existing repository #
```

Comment:
Can we rename this file `skeletonize-existing-repository.md`?  Our (unwritten) standard is to not capitalize filenames (aside from `README` files).

Reply:
Resolved in https://github.com/cisagov/development-guide/pull/32/commits/86b65da847e2443f130512e51b5c35032bd82232